src/jekyll/usage.md

```diff
@@ -154,6 +154,14 @@ The value of `key id` is one of the following:
 * "Name" of the key (e.g. "LAMP/EPFL" in the above example)
 * "Email" of the key (e.g. "lamp@gmail.com" in the above example)
 
-## Publishing Artifacts ##
+## Publishing Artifacts
 
-If you want to published signed artifacts, you must use the new `publish-signed` and `publish-local-signed` tasks.  `sbt-pgp` no longer wires in to the default `publish` and `publish-local` tasks of SBT.  
+If you want to published signed artifacts, you must use the new `publishSigned` and `publishLocalSigned` tasks.
+
+## Skipping publishing
+
+If you want to opt out of publishing set `skip in publish` to `true`.
```

Comment:
> Also, shouldn't we use publishSigned instead of publish?

No, I want the build users to be able to skip publishing without depending on this plugin.